testsuite/features/step_definitions/setup_steps.rb

```diff
@@ -206,16 +206,10 @@
     And I wait until I see the name of "#{host}", refreshing the page
     And I follow this "#{host}" link
     And I wait until I see "System Status" text
+    And I wait 180 seconds until the event is picked up and #{DEFAULT_TIMEOUT} seconds until the event "Apply states" is completed
```

Comment:
we should have used "at most" in this step's name...

Reply:
As you can see, I did not change these steps, I just removed the if statement for the traditional stack. So, I prefer to don't do any change on these steps on this PR.